### ISSUE TITLE
Arrow down in results view has unexpected step

### ISSUE THREAD

**Issue by Biepa** [qarmin/czkawka#1098]
- Czkawka version: 6.1.0 gui
- OS version: Windows 11

I'm not 100% sure if this would be considered a bug, but it felt more like a bug to me than a feature request.

After having done a search (similar images) I click into the resuls pane. Select the first one with my mouse and then go through the pictures with the "arrow down" key and select some of the images with spacebar. The reason for me report is, that when having selected the last image in a group and pressing "arrow down", the selection goes away somehow. I can press down again to get to the first item of the next group, but there is no real reason to not go there directly. So to go from one group to another it's always needed to press arrown down twice, which is a lot when going trough a huge amount of images.

All I've done here is pressing "arrow down".

https://github.com/qarmin/czkawka/assets/25102748/6768fb79-8b88-4d03-9e12-722b970775c0

My request: Jump from the last of a group to the first of the next group immediately.

**Comment by AlexFolland** [comment]
Agreed.  Improving this for the keyboard-driven interface for the process of deleting duplicates would really improve things, along with being able to toggle the checkbox in a row with the space bar.

**Comment by Biepa** [comment]
> Agreed. Improving this for the keyboard-driven interface for the process of deleting duplicates would really improve things, along with always being able to toggle the checkbox in a row with the space bar.

Interestingly, space for selecting doesn't work directly. But after selecting one object in the checkbox with your mouse, then you also can use space for it.

**Comment by qarmin** [comment]
Down arrow is expected to work that since I'm using default gtk behavior and changing it is hard/impossible.
Focus want to change into disabled/header row, but since it is disable do not show selection

Such behavior can be modified quite easily with slint, so probably there will be implemented but not with gtk frontend,

**Comment by qarmin** [comment]
Such behavior is implemented in Krokiet
In czkawka gtk it is impossible, so closing as wont-fix
